src/lib/redux/siteData.ts

```diff
@@ -34,4 +34,10 @@ export const updateTariffPlanInRedux = (tariffPlan: Partial<TariffPlanRecord>) =
 
 export const updateSiteInRedux = (siteUpdates : Partial<SiteRecord>) => {
   store.dispatch(updateSite(siteUpdates))
-}
\ No newline at end of file
+}
+
+// Rounds number to at most 2 decimal points
+// And converts to string
+export const round = (number: number) => {
```

Comment:
Can we actually use this function for all of the numbers we display? We should uniformally use this function (maybe it'd be good to place this in another place like put it in a new `utils` file.


we almost might just be able to use this https://www.w3schools.com/jsref/jsref_tofixed.asp `toFixed` function instead of creating a new function (we might have to watch out for number vs string types though)

Reply:
+1 on moving this to a new more general utils file

Reply:
But if you do stick with this helper rather than `toFixed`, I don't think it's immediately intuitive that `round` returns a string -- maybe rename to something like `roundToString`?